guardian/archivehunter
# fix for "uncaught referenceerror: process is not defined" breaking the UI

## What does this change?
Applies solution at https://stackoverflow.com/a/64553486/2840056 to fix the "process is not defined" error preventing load

## How to test
deploy to dev. you should see the app load correctly, instead of a white screen which we got before

## How can we measure success?
can use the ui
